modules/python/python-dest.c

```diff
@@ -159,7 +160,7 @@ _dd_py_invoke_bool_method_by_name_with_args(PythonDestDriver *self, const gchar
                                                   self->super.super.super.id);
 }
 
-static gboolean
+static gboolean G_GNUC_UNUSED
```

Comment:
`_dd_py_invoke_bool_method_by_name()` is not used anymore. Can we remove it, please? This is actually boilerplate copy-pasted to every Python plugin :(.

Reply:
I left it intentionally because I felt it just happened that all users disappeared from the code. But there is an equivalent chance that in the future there will be some users for it. It is a well written code that worked us well so far.

Still, if you insist, I can remove it.